modules/vba_documents/app/sidekiq/vba_documents/upload_upstream_processing_error_batch.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# Job polls EMMS for status updates for UploadSubmission that hit an EMMS internal procession error.  EMMS seems
+# to have re-processing capacitity and some EMMS errors self recover
+
+require 'sidekiq'
+
+module VBADocuments
+  class UploadUpstreamProcessingErrorBatch
+    include Sidekiq::Job
+
+    # No need to retry since the schedule will run this every hour
+    sidekiq_options retry: false, unique_for: 1.hour
+
+    BATCH_SIZE = 100
```

Comment:
I think it's all right to batch these since we can reuse the `VBADocuments::UploadStatusUpdater` job, but it's unlikely that we'll go over the batch of 100, since on a normal day we have 0-2 submissions with these types of errors and we're only retrying for 14 days.